cves/kernel/CVE-2019-20054.yml

```diff
@@ -1,203 +1,182 @@
-    improving the automated tests?
 
-    For code: and fix: - your answer should be boolean.
+    This can include:
+      * signing off on a commit message
+      * mentioning a discussion with a colleague checking the work
+      * upvoting a solution on a pull request
 
-    For the code_answer below, look not only at the fix but the surrounding
-    code near the fix in related directories and determine if and was there were
-    unit tests involved for this subsystem.
+    Answer must be true or false.
 
-    For the fix_answer below, check if the fix for the vulnerability involves
-    adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
-discovered:
+    Write a note about how you came to the conclusions you did, regardless of what your answer was.
+bounty:
+  amt:
+  url:
+  announced:
+lessons:
+  yagni:
+    note:
+    applies:
   question: |
-    How was this vulnerability discovered?
+    Are there any common lessons we have learned from class that apply to this
+    vulnerability? In other words, could this vulnerability serve as an example
+    of one of those lessons?
 
-    Go to the bug report and read the conversation to find out how this was
-    originally found. Answer in longform below in "answer", fill in the date in
-    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
-    employee (you can tell from their email address). If it's clear that the
-    vulenrability was discovered by a contest, fill in the name there.
+    Leave "applies" blank or put false if you did not see that lesson (you do
+    not need to put a reason). Put "true" if you feel the lesson applies and put
+    a quick explanation of how it applies.
 
-    The automated, contest, and developer flags can be true, false, or nil.
+    Don't feel the need to claim that ALL of these apply, but it's pretty likely
+    that one or two of them apply.
 
-    If there is no evidence as to how this vulnerability was found, then please
-    explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
-autodiscoverable:
-  instructions: |
-    Is it plausible that a fully automated tool could have discovered
-    this? These are tools that require little knowledge of the domain,
-     e.g. automatic static analysis, compiler warnings, fuzzers.
+    If you think of another lesson we covered in class that applies here, feel
+    free to give it a small name and add one in the same format as these.
+  serial_killer:
+    note:
+    applies:
+  complex_inputs:
+    note:
+    applies:
+  distrust_input:
+    note:
+    applies:
+  least_privilege:
+    note:
+    applies:
+  native_wrappers:
+    note:
+    applies:
+  defense_in_depth:
+    note:
+    applies:
+  secure_by_default:
+    note:
+    applies:
+  environment_variables:
+    note:
+    applies:
+  security_by_obscurity:
+    note:
+    applies:
+  frameworks_are_optional:
+    note:
+    applies:
```

Comment:
These are all empty. I am not sure if that was intentional.

Reply:
This was really just bad error handling so I don't think any of these really apply